common/config/rush/.pnpmfile.cjs

```diff
@@ -1,29 +1,49 @@
 function readPackage(pkg) {
-  if ((pkg.name == "typedoc" || pkg.name == "@microsoft/api-extractor") && pkg.dependencies && pkg.dependencies["typescript"])
+  if (
+    (pkg.name == "typedoc" || pkg.name == "@microsoft/api-extractor") &&
+    pkg.dependencies &&
+    pkg.dependencies["typescript"]
+  ) {
     pkg.dependencies["typescript"] = "~4.4.0";
+  }
+
   // Hacky mess: For external packages to this monorepo that have peer dependencies on packages
   // in this repo, we need to do some magic in order to get the peerDeps to point to a correct
   // version of the packages. Update the pkg.json real dependency list to
   // Note that these dependencies are only ever allowed for testing purposes and should not be the
   // dependency of any published packages.
-  else if (pkg.name == "@itwin/imodels-access-backend") {
+
+  // https://github.com/iTwin/imodels-clients
+  if (pkg.name == "@itwin/imodels-access-backend") {
```

Comment:
one last nitpick: why did you change these to different if/else blocks instead of keeping the ifelse syntax? The `pkg.name` is only going to ever be valid for one of these names so if it's already matched one no need to continue checking.

Reply:
you're right, just a minor personal stylistic preference, thought it was better to create a new if/else block for new repos, but will revert.

Reply:
thanks :)